src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/Inputs/SARIF1020.ReferToFinalSchema_Valid.sarif

```diff
@@ -0,0 +1,39 @@
+{
+  "$schema": "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.5.json",
+  "version": "2.1.0",
+  "runs": [
+    {
+      "tool": {
+        "driver": {
+          "name": "Sarif Functional Testing"
+        }
+      },
+      "results": [
```

Comment:
>results [](start = 7, length = 7)

This test doesn't depend on the contents of "results", so replace it with an empty array: "results": []